### PR TITLE
W-10591752 resource allocations for BYOK

### PR DESCRIPTION
**Pull request by IsaacEldridge**
@kmendiratta-crm This PR is to address the missing resource allocation page for BYOK. Since you indicated the pages were basically same save for a few minor changes, I created a partial file so we could reuse the content between both BYOK and appliance. 

Can you please look at `include-deploy-resource-allocation.adoc` and tell me which tagged sections **are not** applicable to BYOK. Thanks.

### CODE REVIEW

File: modules/ROOT/pages/_partials/include-deploy-resource-allocation.adoc [hunk 1]
```diff
@@ -0,0 +1,201 @@
+// tag::description[]
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/_partials/include-deploy-resource-allocation.adoc:1]: @kmendiratta-crm  this is the reusable page. Which sections aren't applicable to BYOK?

File: modules/ROOT/pages/_partials/include-deploy-resource-allocation.adoc [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+
+To check core CPU usage:
+
+. From the Ops Center navigation, click *Monitoring*.
+. From the dashboard dropdown list, select *CPU Core Usage*. 
+. From the time picker, select your date range. 
++
+You might see multiple data points for a certain time frame depending on data granularity. To get an accurate assessment of core CPU usage against licensing, look for the highest number within the selected time frame. 
+// end::cpuLicensedCores[]
+// tag::vcpuAllocation[]
```
Comment (kmendiratta-crm) [inline modules/ROOT/pages/_partials/include-deploy-resource-allocation.adoc]: Ops center monitoring is only available on gravitational appliances. For BYOK customers typically implement/leverage CPU/memory monitoring solutions from the respective cloud providers.

File: modules/ROOT/pages/_partials/include-deploy-resource-allocation.adoc [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+
+** 500 or more simultaneous active connections.
+** Security policies are enabled.
+
+[NOTE]
+These are general guidelines, and individual environments might require adjustments.
+
+// end::loadBalancerMemoryAllocation[]
+// tag::internalLoadBalancer[]
+== Internal Load Balancer
```
Comment (kmendiratta-crm) [inline modules/ROOT/pages/_partials/include-deploy-resource-allocation.adoc:156]: Internal Load Balancer (Edge) is only applicable on gravitational appliances. Does not apply to BYOK

Comment (IsaacEldridge) [reply modules/ROOT/pages/_partials/include-deploy-resource-allocation.adoc:156]: ok

**Review by sarathecoubian — APPROVED** [review]

**Pull request merged**